From 92afa2d97136c65b3585cb172c3f3198a27acaf7 Mon Sep 17 00:00:00 2001
From: Colin Ian King <colin.i.king@gmail.com>
Date: Wed, 9 Mar 2022 16:41:16 +0000
Subject: [PATCH 029/471] ASoC: atmel: mchp-pdmc: Fix spelling mistake
 "microchopnes" -> "microphones"

There is a spelling mistake in a dev_info message. Fix it.

Signed-off-by: Colin Ian King <colin.i.king@gmail.com>
Reviewed-by: Codrin Ciubotariu <codrin.ciubotariu@microchip.com>
Link: https://lore.kernel.org/r/20220309164116.178685-1-colin.i.king@gmail.com
Signed-off-by: Mark Brown <broonie@kernel.org>
---
 sound/soc/atmel/mchp-pdmc.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sound/soc/atmel/mchp-pdmc.c b/sound/soc/atmel/mchp-pdmc.c
index 7b737e3f67b7..1be4007875f1 100644
--- a/sound/soc/atmel/mchp-pdmc.c
+++ b/sound/soc/atmel/mchp-pdmc.c
@@ -892,7 +892,7 @@ static int mchp_pdmc_dt_init(struct mchp_pdmc *dd)
 
 	dd->mic_no /= 2;
 
-	dev_info(dd->dev, "%d PDM microchopnes declared\n", dd->mic_no);
+	dev_info(dd->dev, "%d PDM microphones declared\n", dd->mic_no);
 
 	/* by default, we consider the order of microphones in mchp,mic-pos to
 	 * be the same with the channel mapping; 1st microphone channel 0, 2nd
-- 
2.34.1

